HPXMLtoOpenStudio/resources/EPvalidator.rb

```diff
@@ -749,6 +749,7 @@ def self.run_validator(hpxml_doc)
         'EnergyFactor | CombinedEnergyFactor' => zero_or_one,
         'EnergyFactor | CombinedEnergyFactor | ControlType' => zero_or_two,
         'extension/UsageMultiplier' => zero_or_one,
+        'extension/RatedFlowRate' => zero_or_one,
```

Comment:
It's a bit weird to call the flow rate "rated", since it's not really a rated value. Also, would be clearer if someone could describe the dryer as "vented".

So two suggestions:
1. Add an optional 'extension/IsVented' element (boolean); always default to true.
2. If the above element is provided and true, have `extension/VentedFlowRate` (instead of RatedFlowRate) as an optional element.

Reply:
@shorowit For number one, always default to true when the clothes dryer is non-electric?

Reply:
No, default to true for all fuel types. Electric dryers can (and often are) vented.